src/contour/ui.template/Terminal.qml

```diff
@@ -59,9 +59,16 @@ ContourTerminal
         stepSize : 1.0 / (vtWidget.session.pageLineCount + vtWidget.session.historyLineCount)
     }
 
-    SoundEffect {
+    AudioOutput {
+        id: bellAudioOutput
+        objectName: "BellAudioOutput"
+    }
+
+    MediaPlayer {
         id: bellSoundEffect
-        source: vtWidget.bellSoundSource
+        objectName: "Bell"
+        source: "qrc:/contour/bell.wav"
+        audioOutput: bellAudioOutput
```

Comment:
Only the `.qml.in` files were supposed to be git-tracked in the `ui.template` folder. You can basically drop the changes here and it'll have no affect.

In a future/other PR we should actually get rid of these. :)